web/cobrands/smidsy/js/assets.js

```diff
@@ -0,0 +1,73 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+var heatmap_style = new OpenLayers.Style({}, {
+    context: {
+        getStrokeColor: function(feature) {
```

Comment:
Two thoughts on these colours:

1. It feels odd for the "almost no incidents here" colour to be almost white (#fff5eb), when the next step down, "absolutely zero incidents", is just the raw underlying grey road (#8c8c8c). It feels like maybe the gradient should go red→#8c8c8c rather than red→#ffffff as it currently is.
1. I wonder whether the top end of the colour scale should be brighter too? If we’re getting to densities of 1 incident per metre, I feel like those points of the map should be *shouting* at you, rather than smouldering away in a muddy dark brown.